File: benchmark/bench.c
/* benchmark/bench.c  — sum loop 0..20,000,000 */
#include <stdio.h>
int main(void) {
    long n = 20000000, i = 0, sum = 0;
    while (i < n) { sum += i; i++; }
    printf("%ld\n", sum);
    return 0;
}

File: benchmark/bench.rs
// benchmark/bench.rs — sum loop 0..20,000,000
fn main() {
    let n: i64 = 20_000_000;
    let mut i: i64 = 0;
    let mut sum: i64 = 0;
    while i < n { sum += i; i += 1; }
    println!("{}", sum);
}
